Wrappers/Python/cil/optimisation/algorithms/PDHG.py

```diff
@@ -222,7 +221,16 @@ class PDHG(Algorithm):
     """
 
     def __init__(self, f, g, operator, tau=None, sigma=None, initial=None, gamma_g=None, gamma_fconj=None, **kwargs):
+        """Initialisation of the PDHG algorithm"""
+        
+        self._theta = kwargs.pop('theta', 1.0)
```

Comment:
just curious what the reasoning was for doing this here - previously you did it in set_up

Reply:
Thanks Laura, the reason for doing it here was to remove the kwargs before passing up to the parent class so it didn't raise an error